apps/rewards/app/components/Content/MyRewards.js

```diff
@@ -1,27 +1,203 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 import styled from 'styled-components'
-
+import {
+  Table,
+  TableHeader,
+  TableRow,
+  TableCell,
+  Text,
+  Button,
+  IdentityBadge,
+  Viewport,
+  theme,
+  IconCheck,
+  IconCross,
+  IconFundraising,
+  IconTime,
+  ContextMenu,
+  ContextMenuItem,
+} from '@aragon/ui'
+import { displayCurrency } from '../../utils/helpers'
+import {
+  AverageRewards,
+  formatAvgAmount,
+  RewardDescription,
+  RewardsTable,
+  NarrowList,
+  NarrowListReward,
+  AmountBadge,
+} from './RewardsTables'
 import { Empty } from '../Card'
+import { provideNetwork } from '../../../../../shared/ui'
+
+const averageRewardsTitles = [ 'My Unclaimed Rewards', 'Year to Date', 'Inception to Date' ]
+// TODO: these need to be actually calculated
+const averageRewardsNumbers = [
+  formatAvgAmount(146, '+$', 'green'),
+  formatAvgAmount(19989.88, '$'),
+  formatAvgAmount(799.87, '$'),
+]
+
+const generateOpenDetails = (reward, openDetails) => () => {
+  console.log('calling openDetails on', reward)
+  openDetails(reward)
+}
+
+const translateToken = (token) => {
+  if (token == 0x0) {
+    return 'ETH'
```

Comment:
This block shouldn't be needed because the ETH (and all other tokens') symbol is included in the App's state via the vault integration.

Reply:
ok, I added some mock vault data and removed the translation - is it sufficient for now?